test/test_torrent.cpp

```diff
@@ -474,7 +491,9 @@ TORRENT_TEST(queue)
 	// test set pos on not existing pos
 	torrents[3].queue_position_set(10);
 	torrents[2].queue_position_set(-10);
+	finished.queue_position_set(-10);
```

Comment:
negative numbers are never valid in calls to this function. I think a more realistic test would be to set it to some valid queue position, and still assert that it remains -1, since it's finished